R/parse.r

```diff
@@ -21,7 +21,7 @@ parse_all.character <- function(x, filename = NULL, allow_error = FALSE) {
   on.exit(options(op), add = TRUE)
 
   if (length(grep("\n", x)))
-    x <- unlist(str_split(x, "\n"), recursive = FALSE, use.names = FALSE)
+    x <- unlist(strsplit(x, "\n", fixed = TRUE), recursive = FALSE, use.names = FALSE)
```

Comment:
It may surprise you, but `str_split(x, "\n")` and `strsplit(x, "\n")` are actually not equivalent.

```
> strsplit("", "\n")
[[1]]
character(0)

> stringr::str_split("", "\n")
[[1]]
[1] ""

> strsplit("\n", "\n")
[[1]]
[1] ""

> stringr::str_split("\n", "\n")
[[1]]
[1] "" ""
```

You may borrow the code from here: https://github.com/yihui/knitr/blob/3d6a6c34958102b0eab1907f10b7d3341ef95899/R/utils.R#L597-L604